src/seedu/addressbook/AddressBook.java

```diff
@@ -258,9 +258,9 @@ private static void processProgramArgs(String[] args) {
         if (args.length >= 2) {
             showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
             exitProgram();
-        }if (args.length == 1) {
```

Comment:
In the original code, there wouldn't have been an overlap in the first place. `args.length` could not possibly trigger more than one of the `if` cases.